modulefiles/GDAS/hercules.lua

```diff
@@ -15,64 +15,64 @@ load("stack-intel-oneapi-mpi/2021.9.0")
 --load("stack-python/3.10.8")
 -- JCSDA has 'jedi-fv3-env/unified-dev', but we should load these manually as needed
 load("cmake/3.23.1")
-load("curl/8.1.2")
+load("curl/8.4.0")
 load("zlib/1.2.13")
-load("git/2.40.0")
+load("git/2.31.1")
 --load("pkg-config/0.27.1")
 load("hdf5/1.14.0")
 load("parallel-netcdf/1.12.2")
 load("netcdf-c/4.9.2")
 load("nccmp/1.9.0.1")
-load("netcdf-fortran/4.6.0")
+load("netcdf-fortran/4.6.1")
 load("nco/5.0.6")
 load("parallelio/2.5.10")
 load("wget/1.21.1")
-load("boost/1.78")
+load("boost/1.83.0")
 load("bufr/12.0.1")
 load("git-lfs/3.1.2")
 load("ecbuild/3.7.2")
-load("openjpeg/2.4.0")
-load("eccodes/2.27.0")
+load("openjpeg/2.3.1")
+load("eccodes/2.32.0")
 load("eigen/3.4.0")
-load("openblas/0.3.19")
-load("eckit/1.24.4")
+load("openblas/0.3.24")
+load("eckit/1.24.5")
 load("fftw/3.3.10")
 load("fckit/0.11.0")
 load("fiat/1.2.0")
 load("ectrans/1.2.0")
-load("atlas/0.35.0")
-load("sp/2.3.3")
+load("atlas/0.35.1")
+load("sp/2.5.0")
 load("gsl-lite/0.37.0")
 load("libjpeg/2.1.0")
 load("krb5/1.20.1")
-load("libtirpc/1.2.6")
+load("libtirpc/1.3.3")
 load("hdf/4.2.15")
 load("jedi-cmake/1.4.0")
 load("libpng/1.6.37")
 load("libxt/1.1.5")
 load("libxmu/1.1.4")
-load("libxpm/3.5.12")
+load("libxpm/4.11.0")
 load("libxaw/1.0.13")
 load("udunits/2.2.28")
-load("ncview/2.1.8")
+load("ncview/2.1.9")
 load("netcdf-cxx4/4.3.1")
-load("py-pybind11/2.8.1")
+load("py-pybind11/2.11.0")
 --load("crtm/v2.4_jedi")
 load("contrib/0.1")
 load("noaatools/3.1")
 load("rocoto/1.3.5")
 
 load("hpc/1.2.0")
-unload("python/3.10.8")
-unload("python/3.9.2")
+unload("python/3.10.13")
+unload("py-numpy/1.22.3")
 load("miniconda3/4.6.14")
 load("gdasapp/1.0.0")
 
-load("boost/1.78.0")
+load("boost/1.83.0")
```

Comment:
Tests on hercules find that lines 71 to 75 are not needed.  These lines should be removed.

Reply:
Done.